index.html

```diff
@@ -1949,56 +1949,573 @@ <h2>
+        <dt>
+          <dfn data-lt="Postal codes">Postal code</dfn>
+        </dt>
+        <dd>
+          The postal code or ZIP code, also known as PIN code in India.
+        </dd>
+        <dt>
+          <dfn>Sorting code</dfn>
+        </dt>
+        <dd>
+          The sorting code as used in, for example, France.
+        </dd>
+        <dt>
+          <dfn>Language code</dfn>
+        </dt>
+        <dd>
+          The language in which the address is provided. It's used to determine
+          the field separators and the order of fields when formatting the
+          address for display.
+        </dd>
+        <dt>
+          <dfn>Organization</dfn>
+        </dt>
+        <dd>
+          The organization, firm, company, or institution at the address.
+        </dd>
+        <dt>
+          <dfn>Recipient</dfn>
+        </dt>
+        <dd>
+          The name of the recipient or contact person at the address.
+        </dd>
+        <dt>
+          <dfn>Phone number</dfn>
+        </dt>
+        <dd>
+          The phone number of the recipient or contact person at the address.
+        </dd>
+      </dl>
+      <section data-dfn-for="PaymentAddress" data-link-for="PaymentAddress">
         <h2>
-          Creating a <code>PaymentAddress</code>
+          <dfn>PaymentAddress</dfn> interface
         </h2>
+        <pre class="idl">
+          [SecureContext, Exposed=(Window)]
+          interface PaymentAddress {
+            [Default] object toJSON();
+            readonly attribute DOMString city;
+            readonly attribute DOMString country;
+            readonly attribute DOMString dependentLocality;
+            readonly attribute DOMString languageCode;
+            readonly attribute DOMString organization;
+            readonly attribute DOMString phone;
+            readonly attribute DOMString postalCode;
+            readonly attribute DOMString recipient;
+            readonly attribute DOMString region;
+            readonly attribute DOMString sortingCode;
+            readonly attribute FrozenArray&lt;DOMString&gt; addressLine;
+          };
+        </pre>
         <p>
-          The steps to <dfn>create a payment address</dfn> from user-provided
-          input are given by the following algorithm.
+          The <a>PaymentAddress</a> interface represents a <a>physical
+          address</a>.
         </p>
-        <ol data-link-for="PaymentAddress">
-          <li>Let <var>address</var> be a new instance of
-          <a>PaymentAddress</a>.
-          </li>
-          <li>Set <var>address</var>.<a>[[\addressLine]]</a> to the result of
-          splitting the user-provided address line into a <a data-cite=
+        <section>
+          <h2>
+            Internal Constructor
+          </h2>
+          <p class="note">
+            This constructor algorithm is used internally by the user agent.
+            It's not available to scripts.
+          </p>
+          <p>
+            The steps to <dfn data-lt=
```

Comment:
This is only called from one place, right? I guess it's a separate algorithm because you plan to add a constructor soon? If so make sure to call it out in the commit message, and perhaps in the spec (although that spec call out would disappear shortly).